experiment/measurer.py

```diff
@@ -588,15 +591,21 @@ def set_up_coverage_binary(benchmark):
     coverage_binaries_dir = build_utils.get_coverage_binaries_dir()
     benchmark_coverage_binary_dir = coverage_binaries_dir / benchmark
     if not os.path.exists(benchmark_coverage_binary_dir):
-        os.mkdir(benchmark_coverage_binary_dir)
+        try:
+            os.mkdir(benchmark_coverage_binary_dir)
+        except Exception as err:
+            logger.warning(str(err))
     archive_name = 'coverage-build-%s.tar.gz' % benchmark
     archive_filestore_path = exp_path.gcs(coverage_binaries_dir / archive_name)
     filestore_utils.cp(archive_filestore_path,
                        str(benchmark_coverage_binary_dir))
     archive_path = benchmark_coverage_binary_dir / archive_name
-    tar = tarfile.open(archive_path, 'r:gz')
-    tar.extractall(benchmark_coverage_binary_dir)
-    os.remove(archive_path)
+    try:
```

Comment:
It's hard for me to beleive that all of these exceptions you catch only happen when running locally and cause things to break but don't happen when I run it on the cloud.

Reply:
+1 not clear why we catch exceptions here and turn them into warnings

Reply:
Please see https://github.com/google/fuzzbench/pull/444#discussion_r441080002

Reply:
Please try to investigate why these functions are being called once per trial instead of once per benchmark as intentended (log the benchmarks on 585).